src/driver/postgres/PostgresDriver.ts

```diff
@@ -537,12 +537,20 @@ export class PostgresDriver implements Driver {
             if (columnMetadata.isArray) {
                 // manually convert enum array to array of values (pg does not support, see https://github.com/brianc/node-pg-types/issues/56)
                 value = value !== "{}" ? (value as string).substr(1, (value as string).length - 2).split(",") : [];
-                // convert to number if that exists in poosible enum options
+                // Remove extraneous delimiting quotes around enum strings with spaces.
+                value = value.map((val: string) => {
+                    // Returns an empty string if the length is negative.
+                    const stripped = val.substr(1, val.length - 2);
```

Comment:
This code isn't obvious to me. As I understood you are trying to remove quotes around enum strings by this code. But what if there are no quotes around? Wouldn't it remove regular characters from it?

Reply:
Seconding this.